src/test/java/api/BoundWithTests.java

```diff
@@ -222,6 +222,32 @@ public void canRetrieveBoundWithItemByHoldingsRecordId() throws InterruptedExcep
   }
 
   @Test
+  public void canFetchBoundWithItemWithManyParts() throws InterruptedException, TimeoutException, ExecutionException
+  {
+    IndividualResource instance1 = instancesStorageClient.create( InstanceSamples.smallAngryPlanet( UUID.randomUUID() ).put("title", "Instance 1") );
+    IndividualResource holdings1 = holdingsStorageClient.create(new HoldingRequestBuilder()
+      .forInstance(instance1.getId()).permanentlyInMainLibrary().withCallNumber( "HOLDINGS 1" ));
```

Comment:
I'm curious, where does the (sometimes) putting spaces around parameters in brackets come from?

Reply:
Probably from a previous set of IDE code formatting rules.